packages/eventual-send/src/index.d.ts

```diff
@@ -1,61 +1,69 @@
+// @ts-check
 /* eslint-disable no-shadow,no-use-before-define,no-var,vars-on-top */
 // Type definitions for eventual-send
 // TODO: Add jsdocs.
 
-type Property = string | number | symbol;
+export type Callable = (...args: any[]) => any;
```

Comment:
I don't follow why we need `Callable` instead of `Function`. ??

If we do, should `FunctionOnly` and `RemoteFunction` also change?

Reply:
The problem is that `typeof x extends Function` does not mean that `typeof x === 'function'`, but `typeof x extends Callable` does.

We rely on `typeof x === 'function'` deep in the bowels of the `E` proxy maker.

Reply:
I understood @turadg's question as whether `FunctionOnly` or `RemoteFunction` are the right names, or if it should be `CallableOnly` and `RemoteCallable`. I think it's fine to keep `Callable` internally to help with inference, and `Function` as the external naming, as people are more familiar with that terminology.